private/bufpkg/bufimage/bufimagebuild/builder.go

```diff
@@ -268,6 +272,32 @@ func (b *builder) warnInvalidImports(
 	return nil
 }
 
+var (
```

Comment:
global vars always go at the top of a file (but in this case, they will be deleted)